pkgs/by-name/en/encore/package.nix

```diff
@@ -0,0 +1,59 @@
+{ fetchFromGitHub
+, buildGoModule
+, callPackage
+, makeWrapper
+, lib
+, ...
+}:
+buildGoModule rec {
+  pname = "encore";
+  version = "1.39.0";
+
+  src = fetchFromGitHub {
+    name = "${pname}-source";
+    owner = "encoredev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-71vzo52vV4VilBnLZxIWDKIY08rfTavMZ57qgi4pip8=";
+  };
+
+  doCheck = true;
+
+  subPackages = [
+    "cli/cmd/encore"
+    "cli/cmd/git-remote-encore"
+    "cli/cmd/tsbundler-encore"
+  ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  CGO_ENABLED = 1;
+
+  postInstall =
+    let
+      goEncore = callPackage ./go.nix { };
+    in
+    ''
+      mkdir -p $out/share/runtimes
+      cp -r $src/runtimes/* $out/share/runtimes
+
+      ln -s ${goEncore}/bin/* $out/bin
```

Comment:
Can you check whether it is possible to use `lib.getBin` here, please?
```suggestion
      ln -s ${lib.getBin goEncore}/* $out/bin
```

Reply:
Is not possible like that.

![image](https://github.com/user-attachments/assets/673854a4-b6b2-4bd0-94ad-da4ddccef53f)

![image](https://github.com/user-attachments/assets/79006a73-1200-46e0-b8f6-527dde7b317f)

Reply:
Thanks for trying this out 👍 Not sure I understand why it doesn't work, but I'll leave that for another day 🙂